yearclass/src/main/java/com/facebook/device/yearclass/YearClass.java

```diff
@@ -100,7 +100,8 @@ private static int getNumCoresYear() {
     if (cores < 1) return CLASS_UNKNOWN;
     if (cores == 1) return CLASS_2008;
     if (cores <= 3) return CLASS_2011;
-    return CLASS_2012;
+    if (cores <= 7) return CLASS_2012;
+    return CLASS_2014;
```

Comment:
Can you separate this chunk into its own diff? Changing the heuristic is scary to do without a data study.